style.css

```diff
@@ -743,3 +747,677 @@ button:active {
+    margin: 0 0;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    margin-left: 12px;
+  }
+
+  .year {
+    color: #6b778c;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    margin-left: 12px;
+  }
+
+  .desc-title {
+    margin-left: 24px;
+    margin-bottom: 24px;
+  }
+
+  .primary-text2 {
+    flex: none;
+    flex-grow: 0;
+    order: 1;
+    margin: 12px 0;
+    color: #344563;
+    position: static;
+    left: 0;
+    top: 92px;
+    height: 90px;
+    width: 295px;
+    font-family: 'Poppins', sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    font-style: normal;
+    line-height: 24px;
+  }
+
+  .tags {
+    margin: 0;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 8px;
+    width: 316px;
+    height: 24px;
+    list-style-type: none;
+  }
+
+  .tag {
+    float: right;
+    text-align: center;
+    background-color: #ebebff;
+    border-radius: 8px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+    padding: 5px;
+    gap: 8px;
+    margin-left: 0;
+  }
+
+  .tag2 {
+    margin-top: 48px;
+  }
+
+  .btn {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    position: absolute;
+    width: 124px;
+    height: 48px;
+    margin-left: 0;
+    margin-top: 54px;
+    top: calc(50% - 48px / 2);
+    background: #fff;
+    border: 1px solid #7f8cff;
+    box-sizing: border-box;
+    border-radius: 8px;
+    order: 3;
+  }
+
+  .action {
+    position: static;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+    flex: none;
+    flex-grow: 0;
+    text-decoration: none;
+  }
+
+  .btn:hover {
+    background: #2230d2;
+    border: 1px solid #7f8cff;
+  }
+
+  .btn:active {
+    background: #0416b6;
+    color: white;
+  }
+
+  .btn:visited {
+    background: #2230d2;
+    border: 1px solid #7f8cff;
+  }
+
+  .btn:disabled {
+    background: #e5e5e5;
+    border: 1px solid;
+  }
+
+  .action:hover {
+    color: #fff;
+  }
+
+  .action:visited {
+    color: #fff;
+  }
+
+  .action:disabled {
+    color: #5e6c84;
+  }
+
+  .about {
+    position: absolute;
+    flex-direction: row;
+    gap: 24px;
+    align-items: center;
+    box-sizing: border-box;
+    width: 100%;
+    height: 924px;
+    background: url("images/abt-myself-desktop.png") no-repeat;
+    border-top-right-radius: 80px;
+    top: 3600px;
+  }
```

Comment:
- Please, kindly add a `background-size: cover` to fix the issue with the about section not taking the full width as seen below in your design:

![spacee_behind](https://user-images.githubusercontent.com/59397159/186716522-5747b777-fe78-4a4c-a79e-cd9f7b3d2647.png)

Reply:
Thank you for the suggestion, it is resolved